sklearn/manifold/mds.py

```diff
@@ -149,70 +151,73 @@ def smacof(similarities, metric=True, n_components=2, init=None, n_init=8,
     3. Compute the Guttman Transform
     4. Iterate 2 and 3 until convergence.
 
-    The nonmetric algorithm adds a monotonic regression steps before computing
+    The nonmetric algorithm adds a monotonic regression step before computing
     the stress.
 
     Parameters
     ----------
-    similarities : symmetric ndarray, shape (n_samples, n_samples)
-        similarities between the points
+    dissimilarities : ndarray, shape (n_samples, n_samples)
+        Pairwise dissimilarities between the points. Must be symmetric.
 
     metric : boolean, optional, default: True
-        compute metric or nonmetric SMACOF algorithm
+        Compute metric or nonmetric SMACOF algorithm.
 
     n_components : int, optional, default: 2
-        number of dimension in which to immerse the similarities
-        overridden if initial array is provided.
+        Number of dimensions in which to immerse the dissimilarities. If an
+        ``init`` array is provided, this option is overridden and the shape of
+        ``init`` is used to determine the dimensionality of the embedding
+        space.
 
-    init : {None or ndarray of shape (n_samples, n_components)}, optional
-        if None, randomly chooses the initial configuration
-        if ndarray, initialize the SMACOF algorithm with this array
+    init : ndarray, shape (n_samples, n_components), optional
```

Comment:
can you add: `default: None`. It is unclear from the description that None is the default (and that None is an acceptable value for this argument)